install_bootstrap.md

```diff
@@ -0,0 +1,159 @@
+## What you will learn in this section
+
+This section will help you understand and install Bootstrap on your system.
+
+The first thing you need to do is access the getting started page on getBootstrap. Here you will notice three different download options depending on your level and skill set.
+
+If you’re completely new to Bootstrap, you’ll want to stick with the basic Bootstrap download. This contains compiled and minified CSS, JavaScript, and fonts. No docs or original source files are included.
```

Comment:
This paragraph will make beginner confuse with minified word. Lets remove minified at this stage.